app/donglet/app-g031.toml

```diff
@@ -110,6 +111,7 @@ address = 0b1010_000
 device = "at24csw080"
 description = "Gimlet Fan VPD"
 removable = true
+refdes = ["J3", "J5"]
```

Comment:
I think this should be U1, based on https://github.com/oxidecomputer/hardware-gimlet-fan-vpd/releases/tag/gimlet-fan-vpd_C_913-0000022_rev2_tapeout

Reply:
I think on Donglet there's a separate PMOD adapter board that the vpd eeproms are plugged into, based on https://github.com/oxidecomputer/hardware-pmod_vpd_prog/blob/main/hardware-pmod_vpd_prog.pdf

Now that we can accept arbitrarily long refdes paths here, I think it could be `J3/J5/U1` and similar for the other eeproms on donglet.